backend/spec/export_ead3_spec.rb

```diff
@@ -209,9 +209,11 @@ def archival_object(index)
 
       as_test_user("admin", true) do
         load_export_fixtures
-        @doc = get_xml("/repositories/#{$repo_id}/resource_descriptions/#{@resource.id}.xml?include_unpublished=true&include_daos=true&ead3=true")
```

Comment:
I think we should keep a test that checks that the default behavior (the one that is currently happening on master) also continues to happen on these changes if no `include_uri_unitids` parameter is present in the request.